### PR TITLE
MultiVM for data-analytics, update hadoop to 2.9.2

### PR DESCRIPTION
**Pull request by komal1204**

### CODE REVIEW

File: commons/java/openjdk8/Dockerfile [hunk 1]
```diff
@@ -2,5 +2,3 @@ FROM ubuntu:18.04
 
 RUN apt-get update -y && apt-get install -y --no-install-recommends openjdk-8-jdk-headless \
 	&& rm -rf /var/lib/apt/lists/*
-
-ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
```
Comment (neo-apz) [inline commons/java/openjdk8/Dockerfile:6]: I believe this change is not needed for this PR, right?
If yes, can you please revert the changes to this file?
Thanks!

**Comment by neo-apz** [comment]
We will merge this PR after merging #248 (Java 11).

**Comment by neo-apz** [comment]
@alok123t @komal1204 please add the required line to travis file for building the data-analytics image for version 4.0.0.

**Comment by alok123t** [comment]
@neo-apz updated travis

File: docs/benchmarks/data-analytics.md [hunk 2]
```diff
@@ -5,45 +5,39 @@
...
 
 ## Images ##
 
 To obtain the images:
 
 ```bash
-$ docker pull cloudsuite/hadoop
-$ docker pull cloudsuite/data-analytics
+$ docker pull cloudsuite/hadoop:2.9.2
+$ docker pull cloudsuite/data-analytics:3.0.0
```
Comment (neo-apz) [inline docs/benchmarks/data-analytics.md:16]: One more thing, this should be version 4 not version 3.
